### DISCUSSION TITLE
Release 0.20.0 is missing the windows version

### DISCUSSION THREAD

**Discussion by nils-a** [extrawurst/gitui#1106] [General]
I stumbled upon the fact that [release 0.20.0](https://github.com/extrawurst/gitui/releases/tag/v0.20.0) is missing the windows version.

Is that a mistake or was this done on purpose?

**Comment by extrawurst** [comment]
It was a bug in the CD: https://github.com/extrawurst/gitui/commit/acb7e5269677a1832ddc5d644e98b2361584e1c0

I discovered a severe performance regression in 0.20 so there will be a 0.20.1 soon which will be deployed to windows aswell then.

see https://github.com/extrawurst/gitui/issues/1102
